src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -29,6 +29,10 @@ Copyright (c) .NET Foundation. All rights reserved.
   <UsingTask TaskName="GetAssemblyVersion" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
   <UsingTask TaskName="GenerateSatelliteAssemblies" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
 
+  <ItemGroup Condition="'$(BuildWithFrameworkRoslyn)' == 'true' and '$(MicrosoftNetCompilersToolsetPackageVersion)' != ''">
```

Comment:
We also don't appear to mention the name "Roslyn" much as part of official property names. It's there in some APICompat code, but minimally otherwise. Should we be using something other than the original codename?

Reply:
Sorry I was late getting back to this. Do you have a suggestion for a better property name?